lib/chezmoi/target_state_test.go

```diff
@@ -137,6 +137,7 @@ func TestTargetStatePopulate(t *testing.T) {
 						contents:   []byte("bar"),
 					},
 				},
+				Scripts: map[string]*Script{},
```

Comment:
I don't think these empty maps are needed. They _should_ compare OK if they are left uninitialized (i.e. `nil`).

Reply:
I tested it, they are required.
```
--- FAIL: TestTargetStatePopulate/script_force_executable (0.00s)
        target_state_test.go:530: ts.Populate(&{PathFS:{fs:0x17858e0 path:/var/folders/7b/_plxss852cn6ch8nfc4djxy40000gn/T/go-vfs-vfst228732403} tempDir:/var/folders/7b/_plxss852cn6ch8nfc4djxy40000gn/T/go-vfs-vfst228732403 keep:false}) diff:
            modified: .Scripts = map[string]*chezmoi.Script{}
```

Reply:
This might have changed following https://github.com/twpayne/chezmoi/pull/234. Could you do a quick re-check?